services/report/__init__.py

```diff
@@ -546,7 +547,7 @@ async def _do_build_report_from_commit(self, commit) -> Report:
         report = self.get_existing_report_for_commit(commit)
         if report is not None:
             return report
-        return await self.create_new_report_for_commit(commit)
+        return await self.create_new_report_for_commit(commit, None)
```

Comment:
Just for readability I think passing the args as `(commit, report_code=None)` would be nice